spdx-utils/src/main/kotlin/Extensions.kt

```diff
@@ -26,6 +26,44 @@ import java.util.EnumSet
  */
 operator fun <E : Enum<E>> EnumSet<E>.plus(other: EnumSet<E>): EnumSet<E> = EnumSet.copyOf(this).apply { addAll(other) }
 
+/**
+ * Create an [SpdxLicenseExceptionExpression] from this [SpdxLicenseException].
+ */
+fun SpdxLicenseException.toExpression() = SpdxLicenseExceptionExpression(id)
+
+/**
+ * Create an [SpdxExpression] by concatenating [this] and [other] using [SpdxOperator.AND].
```

Comment:
I prefer the existing sentence, especially because "...by concatenating this or an other" in the comment below sounds confusing. But I have added the link to [SpdxLicense] because only [this] did not work. I also like how the result created by dokka looks like:

```
and
infix fun SpdxLicense.and(other: SpdxLicense): SpdxCompoundExpression
infix fun SpdxLicense.and(other: SpdxExpression): SpdxCompoundExpression

Create an SpdxExpression by concatenating this and other using SpdxOperator.AND.
```

As both extensions get grouped together it makes sense not to add a link to [other].